discovery-provider/src/queries/get_health.py

```diff
@@ -197,7 +199,17 @@ def get_health(args, use_redis_cache=True):
     user_balances_age_sec = get_elapsed_time_redis(
         redis, user_balances_refresh_last_completion_redis_key
     )
-
+    num_users_in_balance_refresh_queue = len(
```

Comment:
soon, we should start using a redis.pipeline() for this. That's how the monitors `get_monitors` fn works for redis.